e2e/cypress/integration/dataset.spec.js

```diff
@@ -40,11 +41,12 @@ describe('Dataset', () => {
             formData.set('name', "test-resource-1");
             formData.set('resource_type', "CSV");
             formData.set('format', "CSV");
-            cy.form_request('POST', 'http://app:5000/api/action/resource_create', formData, function (response) {
+            cy.form_request('POST', '/api/action/resource_create', formData, function (response) {
                 expect(response.status).to.eq(200);
             });
         });
-    })
+        cy.visit('/dataset');
+    });
```

Comment:
Do we still want this line here?

Reply:
It was to make sure the fixture was captured in the right test.  Without this line, the fixture was just run as part of the next test.  I don't know if we can write the test differently to capture the fixture properly.